src/management.jl

```diff
@@ -0,0 +1,79 @@
+function get_backtrace(ex)
+    v = IOBuffer()
+    Base.showerror(v, ex, catch_backtrace())
+    return v.data |> copy |> String
+end
+
+function get_log_level(l)
+    log_level_str = lowercase(l)
+
+    (log_level_str == "debug") ? Logging.Debug :
+    (log_level_str == "info")  ? Logging.Info  :
+    (log_level_str == "warn")  ? Logging.Warn  : Logging.Error
+end
+
+function status_monitor(stop_file::AbstractString,
+                        zsock::MessageSocket,
+                    )
+    while isopen(zsock.sock)
+        sleep(5)
+        flush(stdout); flush(stderr);
+        # stop server if stop is requested
+        if isfile(stop_file)
+            @warn("Server stop requested.")
+            flush(stdout); flush(stderr)
```

Comment:
Why are these being flushed manually?
That is rarely required.

Reply:
Required when using `nohup`

Reply:
comment as such?